select_statements.sql

```diff
@@ -31,89 +31,60 @@ UPDATE vehicle
 SET stationName = "Linköping station"
 WHERE registrationNumber = "CAG461";
 
-UPDATE vehicle
-SET stationName = "Lund station"
-WHERE registrationNumber = "AAD123";
-
-UPDATE vehicle
-SET stationName = "Lund station"
-WHERE registrationNumber = "AAC223";
-
--- Select all cars
-SELECT v.registrationNumber as Registreringsnummer, vc.name as Fordonskategori, v.stationName as Station
-FROM vehicle as v
-INNER JOIN vehicle_category as vc ON v.vehicleCategoryId = vc.vehicleCategoryId
-ORDER BY Station, FordonsKategori;
-
-
 -- Select all vacant cars for a station
 SELECT v.registrationNumber as Registreringsnummer, vc.name as Fordonskategori
-FROM vehicle as v
+FROM green_rental.vehicle as v
```

Comment:
I removed `green_rental.` from all statements to make them easier to read in the report.